docs/reference/overview.md

```diff
@@ -11,6 +11,7 @@ This section contains general reference material for Camunda 8.
 
 - [Announcements](announcements.md) - Stay up to date on versioning changes, end of maintenance updates, OpenSearch support, and other updates.
 - [Release notes](release-notes.md) - Learn where you can find release notes for components of Camunda 8.
+- [Auto-updates](auto-updates.md) - Some updates are applied to your SaaS clusters automatically.
```

Comment:
"some" reads very abstract in this sentence. Suggestion: "SaaS Clusters auto update configuration."

Reply:
I'd prefer to have the configuration in the console docs, but I'll adjust this sentence. 

> Learn about the SaaS cluster auto-update feature.